frontend/models/Messages.php

```diff
@@ -44,9 +50,22 @@ public function attributeLabels()
             'id' => 'ID',
             'user_id' => 'User ID',
             'message' => 'Message',
+            'task_id' => 'Task ID',
+            'is_mine' => 'Is Mine',
+            'published_at' => 'Published At',
         ];
     }
 
+    /**
+     * Gets query for [[Task]].
+     *
+     * @return \yii\db\ActiveQuery|TasksQuery
+     */
+    public function getTask()
+    {
+        return $this->hasOne(Tasks::className(), ['id' => 'task_id']);
+    }
+
```

Comment:
Вот метод fields, про который я писал. Позволяет добавить в модель поле is_mine динамически.
```suggestion
    public function fields()
    {
        return array_merge(parent::fields(), [
            'is_mine' => function () {
                return (int)Yii::$app->getUser()->getId() === $this->user_id;
            },
        ]);
    }
```
Добавление этого метода позволяет в дополнение к уже существующим в модели полям добавить новое (is_mine) и указать для него значение.

Документация с примером https://yiiframework.com.ua/ru/doc/guide/2/rest-resources/#overriding-fields